remote-release-notes/v1_97.md

```diff
@@ -18,4 +18,8 @@ We are continuing to improve this experience and would love to [hear your feedba
 
 A new setting has been introduced to install extensions into a remote if that extension is already present locally. This mirrors the setting already available in the Dev Containers extension.
 
-Configure with `setting(remote.SSH.defaultExtensionsIfInstalledLocally)`.
\ No newline at end of file
+Configure with `setting(remote.SSH.defaultExtensionsIfInstalledLocally)`.
+
+### Migration path for legacy server users
```

Comment:
Let's add Linux to the title to make it obvious. So

Migration path for legacy Linux server users